core/src/main/scala/kafka/server/DelayedCreatePartitions.scala

```diff
@@ -49,7 +51,7 @@ class DelayedCreatePartitions(delayMs: Long,
                               createMetadata: Seq[CreatePartitionsMetadata],
                               adminManager: ZkAdminManager,
                               responseCallback: Map[String, ApiError] => Unit)
-  extends DelayedOperation(delayMs) {
+  extends DelayedOperation(delayMs) with Logging {
```

Comment:
The `Logging` trait used to be on `DelayedOperation` class. Since I rewrote `DelayedOperation` in Java, I added the `Logging` trait to all these classes.

I expect we'll move these soon, then we'll use `Logger` for them too.